dotcom-rendering/src/components/EditorialButton/EditorialButton.stories.tsx

```diff
@@ -0,0 +1,163 @@
+import { css } from '@emotion/react';
+import {
+	ArticleDesign,
+	ArticleDisplay,
+	ArticlePillar,
+	ArticleSpecial,
+} from '@guardian/libs';
```

Comment:
How will this be affected by #12461 ? Will we do a follow-up PR to switch to the local versions?

Reply:
I'm happy to do a follow-up PR, or update this PR, after we merge #12461